app/controllers/allocations_controller.rb

```diff
@@ -20,10 +20,9 @@ def new
   # rubocop:disable Metrics/LineLength
   def show
     @prisoner = offender(nomis_offender_id_from_url)
-
     allocation = AllocationService.active_allocations_with_pom_detail(@prisoner.offender_no).first
     @pom = PrisonOffenderManagerService.get_pom(active_caseload, allocation.nomis_staff_id)
-
+    @keyworker = Nomis::Keyworker::KeyworkerApi.get_keyworker(active_caseload, @prisoner.offender_no)
```

Comment:
I did not think there was any point putting in a Keyworker service at this stage, but happy to add if preferred

Reply:
Can't imagine we'll need anything else from it at this point. If we do we can consider it then.